### ISSUE TITLE
"Save as screenshot" not available after spawning photo from own inventory

### ISSUE THREAD

**Issue by Reqces** [Neos-Metaverse/NeosPublic#2647]
### Describe the bug?

"Save as screenshot" option within the context menu for photos taken within Neos doesn't appear if the image was spawned out from my inventory.

Taking photos myself still works as it will automatically save as a screenshot for both steam and the default photos location.

### Solutions Tried
- Clearing screenshots directory on both Steam and default location
- Changing screenshot save directory
- Reinstalling Neos
- Changed avatars/worlds
- Ran Neos on a different system machine

### Relevant issues

No other issues with NeosVR.

### To Reproduce

1) Spawn out a photo from your inventory

2) Hold it an open context menu. Option does not appear within radial menu.

### Expected behavior

Having the "Save as screenshot" option appear within radial menu.

### Screenshots

![Screenshot2](https://user-images.githubusercontent.com/87484234/125800790-33a52e5b-9979-46cb-9c6f-e4b77945aee8.png)
"Save as screenshot" not present in radial menu for photo (taken by someone else).

![Screenshot1](https://user-images.githubusercontent.com/87484234/125801251-4c4ade89-e6e2-4138-b0b5-402c231c4a8c.png)
Showing that taking photos still saves automatically but also "Save as screenshot" still not present.

![2021-07-14 21 19 48](https://user-images.githubusercontent.com/87484234/125801117-8bd6a644-17e5-42c7-a9d1-7ceefb5bc0d5.jpg)
Friend showing that "Save as screenshot" appears in their radial menu for photo (taken by someone else).

### How often does it happen?

Always

### Does the bug persist after restarting Neos?

Yes

### Neos Version Number

2021.7.11.1002

### What Platforms does this occur on?

- [ ] Any
- [X] Windows
- [ ] Linux
- [ ] Android / Quest

### Did this work before?

Yes

### If it worked before, on which build?

2021.7.9.1193

### Additional context

I have tried both setting a photo path in the settings menu or leaving it blank, but does not do anything even after restarting Neos.

### Reporters

FreeSpirit#6969 @ Discord | FreeSpirit @ NeosVR

**Comment by Frooxius** [comment]
I cannot replicate this on my end, when another user takes or spawns photo, the option correctly appears.

The option not appearing for you is normal behavior, the screenshot is already implicitly saved when you take it, so having the option appear would be redundant.

Does this persist even if you switch to another avatar? We'd need some more info to narrow this down.

**Comment by Reqces** [comment]
Yes, this persists even if I change to another avatar. I have even tried changing worlds as well.

**Comment by Frooxius** [comment]
Hmm I'm not sure what it could be unfortunately, we'd need some more information on how to replicate this problem or see if it's affecting more users.

**Comment by Reqces** [comment]
I have reinstalled Neos and even tried running it on a completely different system machine with no luck. I could possibly try hopping online and demonstrating live on my end.

Give me a poke Discord and we can schedule a time for me to show you.

**Comment by shiftyscales** [comment]
Hi @Frooxius - I've spoken with FreeSpirit, and got some clarification over their issue.

This case specifically comes up when a user spawns the photo from their own inventory and wishes to re-save it, e.g. as a quick means of exporting it to their PC/categorizing it into their auto-save screenshots folder.

There is a similar case that happens in the form of issue #2562, in which a screenshot imported as a Neos photo can't be saved as a screenshot by the person that imported it, nor does it get automatically saved to their auto-save screenshots folder after import.

**Comment by Frooxius** [comment]
I see thanks! Yeah in that case the behavior is intentional and has been present since the beginning. Since the screenshot is already saved in that case, the option won't appear.

What's the cause-case for this? If you have it already saved, why do you want to save it again? That will result in a duplicate copy being saved and cause potential confusion on what screenshots you have saved and which you haven't.

It might be better to instead use an export option instead. Or potentially we could change the option so it allows re-saving, but indicates that it's already been saved somehow.

**Comment by Reqces** [comment]
How I stumbled upon this case was during vBLFC where I was with some friends and we were sharing photos we took.

I saved the photos that my friends shared with me that they took into my inventory but I haven't exported/saved it as a screenshot because I was heading to bed. My thought process was by saving it into my inventory, I can spawn it in my home world to export but also to display in a photo wall I had created as well as the possibility of sharing it with friends without having to reimport it. So, some of the photos in my inventory were never exported in the first place and I wasn't saving it "again".

Also, this behavior is present if I was running on a different system machine that has none of the photos. This can be an inconvenience if I have the screenshots saved/exported on Computer A but because I'm on Computer B I can no longer resave/export it since it's has already been done on Computer A.
